packages/caver-contract/src/index.d.ts

```diff
@@ -0,0 +1,139 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { Wallet } from '../../..'
+import { Result } from '../../caver-abi/src'
+import { KIP37 } from '../../caver-kct/src'
+import KIP7 from '../../caver-kct/src/kip7'
+import { AbiItem } from '../../caver-utils/src'
+import KeyringContainer from '../../caver-wallet/src'
+
+export interface ContractOptions {
+    address: string
+    jsonInterface: AbiItem[]
+    from: string
+    feePayer: string
+    feeDelegation: boolean
+    feeRatio: number
+    gasPrice: number
+    gas: number
+    data: string
+}
+
+export interface DeployOptions extends SendOptions {
+    data?: string
+    arguments?: any[]
+}
+
+export interface SendOptions {
+    from?: string
+    feeDelegation?: boolean
+    feePayer?: string
+    feeRatio?: number
+    gas?: number | string
+    contractDeployFormatter?: Function
+}
+
+export interface CallOptions {
+    from?: string
+    gasPrice?: string
+    gas?: number
+}
+
+export interface ContractSendMethod extends SendData {
```

Comment:
Can you change name from `ContractSendMethod` to `ContractMethod`?
http://kips.klaytn.com/KIPs/kip-34#contract-abi-kct-layer